validate.lic

```diff
@@ -31,6 +31,18 @@ class DRYamlValidator
       respond("   PASSED")
     end
 
+    respond("")
+    respond("  CHECKING: That character yaml files contain valid YAML.")
+    Dir.glob("./scripts/profiles/#{checkname}-*.yaml") { |file|
+      begin
+        YAML.parse(File.open(file))
+        respond("  #{File.basename(file)} PASSED")
+        # respond("   PASSED")
```

Comment:
I like this PR a lot.

Got an errant commented-out line here. Keep or remove?

Reply:
Removed. Ready to merge.